snapshot/Makefile

```diff
@@ -36,6 +36,18 @@ ifeq ($(VERSION),)
 	VERSION = latest
 endif
 
+ifeq (${BASE_DOCKER_IMAGE_ARM64}, )
+  BASE_DOCKER_IMAGE_ARM64 = "arm64v8/alpine"
```

Comment:
Is this required? I think depending no the machine on which docker is running, the docker pull will automatically pick up the arm image.

Reply:
Yes , it's required. Docker will not pull unless we mention it.

Reply:
Nope. apline is a multi-arch manifest. Depending on which OS is pulling the image, the correct docker image will be pulled. 

As travis runs from arm64 and arm64 machine, the correct architecture images are pulled within travis.